analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -275,7 +277,13 @@ class GoDep(
 
         val repoRoot = Paths.get(gopath.path, "src", importPath).toFile()
 
+        // `go get` should always provide VCS information. However, if it doesn't the importPath can be used to create
```

Comment:
Again, it's not `go get` itself that provides the VCS information. Proposal:

The `processProjectVcs()` function should always be able to deduce VCS information from the working tree created by `go get`. However, if that fails for whatever reason, fall back to guessing VCS information from the `importPath` (which usually resembles a URL).